src/components/navbar.js

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import FuzzySearch from 'fuzzy-search';
+import { selectAllCovidData, searchData } from '../features/covid/covidSlice';
+
+function Navbar() {
+  const initialState = { search: '' };
+  const [search, setSearch] = useState(initialState);
+  const dispatch = useDispatch();
+  // const navigate = useNavigate();
+
+  const covidData = useSelector(selectAllCovidData);
+
+  const handleSubmitSearch = (data) => {
+    if (data) {
+      const searcher = new FuzzySearch(
+        covidData,
+        [
+          'item_id',
+          'deaths',
+          'country',
+          'continent',
+          'capital_city',
+          'confirmed',
+          'abbreviation',
+        ],
+        {
+          caseSensitive: false,
+        },
+      );
+      const result = searcher.search(data);
+      dispatch(searchData(result));
+      // navigate('/');
+      setSearch(initialState);
+    }
+    return false;
+  };
+
+  const continents = [
+    'Africa',
+    'Asia',
+    'Europe',
+    'North America',
+    'Oceania',
+    'South America',
+  ];
+
+  const renderContinents = () =>
+    // eslint-disable-next-line implicit-arrow-linebreak
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.